health-services/project-factory/src/server/api/coreApis.ts

```
@@ -23,25 +25,51 @@
  */
+/**
+ * Searches MDMS data using the v2 API with optional caching.
+ * 
+ * @param criteria - The MDMS criteria used to search.
+ * @param cacheEnabled - Enables cache key header if true.
+ * @returns A promise resolving to MDMS v2 API response.
+ */
 const searchMDMSDataViaV2Api = async (
-  MdmsCriteria: MDMSModels.MDMSv2RequestCriteria
+  criteria: MDMSModels.MDMSv2RequestCriteria,
+  cacheEnabled: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)